server/memphis_helper_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019-2022 The NATS Authors
```

Comment:
please replace those headers to Memphis headers across the entire application. You can find the Memphis headers inside 1 of the memphis files from the staging branch